utils/pipelines/main.py

```diff
@@ -47,10 +47,14 @@ def get_last_assistant_message(messages: List[dict]) -> str:
     return None
 
 
-def get_system_message(messages: List[dict]) -> dict:
```

Comment:
`get_system_message` should return message dict

feel free to create `get_system_message_content` util to meet your needs.